backend/public/index.html

```diff
@@ -2,6 +2,61 @@
 
 <head>
   <title>EXPRESS INDEX</title>
+  <style>
```

Comment:
these should probably be inlined into the react components with styled-components

Reply:
Makes sense, I was intentionally keeping them out of the JSX to keep the core samples as tight as possible. This allows us to keep the structure / navigation independent of what is happening in the samples